content/actions/using-workflows/reusing-workflows.md

```diff
@@ -363,7 +363,7 @@ A reusable workflow may generate data that you want to use in the caller workflo
 If a reusable workflow that sets an output is executed with a matrix strategy, the output will be the output set by the last successful completing reusable workflow of the matrix which actually sets a value.
 That means if the last successful completing reusable workflow sets an empty string for its output, and the second last successful completing reusable workflow sets an actual value for its output, the output will contain the value of the second last completing reusable workflow.{% endif %}
 
-The following reusable workflow has a single job containing two steps. In each of these steps we set a single word as the output: "hello" and "world." In the `outputs` section of the job, we map these step outputs to job outputs called: `output1` and `output2`. In the `on.workflow_call.outputs` section we then define two outputs for the workflow itself, one called `firstword` which we map to `output1`, and one called `secondword` which we map to `output2`.
+The following reusable workflow has a single job containing two steps. In each of these steps we set a single word as the output: "hello" and "world." In the `outputs` section of the job, we map these step outputs to job outputs called: `output1` and `output2`. In the `on.workflow_call.outputs` section we then define two outputs for the workflow itself, one called `firstword` which we map to `output1`, and one called `secondword` which we map to `output2`.  Note that only job-level outputs can be mapped to workflow outputs.  Step-level outputs must first be mapped to job-level outputs as shown below.
```

Comment:
```suggestion
The following reusable workflow has a single job containing two steps. In each of these steps we set a single word as the output: "hello" and "world." In the `outputs` section of the job, we map these step outputs to job outputs called: `output1` and `output2`. In the `on.workflow_call.outputs` section we then define two outputs for the workflow itself, one called `firstword` which we map to `output1`, and one called `secondword` which we map to `output2`.  For more information about setting outputs with `jobs.<job_id>.outputs`, see "[AUTOTITLE](/actions/using-jobs/defining-outputs-for-jobs#overview)."
```

Reply:
I am all for linking cross-references and I would support adding your final sentence (with the link) after my added sentences distinguishing between job- and step-level outputs.  Alternatively, a caveat such as what I drafted could be added to the documentation that you linked instead of (or in addition to) being added here.   

Here's what I found unclear: the example on this page references `${{ steps.step1.outputs.firstword }}` within the `jobs.<job_id>.outputs` mapping, and then uses that job output in the `on.workflow_call.outputs` mapping.  Why not simplify as follows:

```
on:
  workflow_call:
    # Map the workflow outputs to job outputs
    outputs:
      firstword:
        description: "The first output string"
        value: ${{ jobs.example_job.steps.step1.outputs.test }}
```

Clearly step-level outputs _exist_ as they are used in the example mapping to generate job-level outputs... one could imagine cases where the job-level output might involve some transformation or manipulation of step-level outputs, but since they are merely passed through here, why not reference them directly?  The answer is that you just can't do it that way -- the `workflow_call` trigger can see job outputs, but it cannot see the step outputs inside the job definition.  However, the current documentation does not make this clear.  The runner logs ALSO do not make this clear, as `${{ jobs.example_job.steps.step1.outputs.test }}` simply evaluates to a null value and gets substituted in as `''`.

My proposed text, "Note that only job-level outputs can be mapped to workflow outputs.  Step-level outputs must first be mapped to job-level outputs as shown below", is intended to clarify this specific issue.

Reply:
I see what you're saying @APCBoston, thanks for the clarification! That was helpful. Your analysis makes sense to me, and I noticed that our [`on.workflow_call.outputs`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_calloutputs) documentation does say: 

> The `value` must be set to the value of an output from a job within the called workflow.

This implies that step-level outputs are not available to the caller workflow, but it doesn't explicitly describe the dynamic you're describing. 

So, inspired by your original suggestion, I think we can go back to explicitly calling this out with the language from the Workflow syntax for GitHub Actions article. 

What do you think of something like this? 

```suggestion
The following reusable workflow has a single job containing two steps. In each of these steps we set a single word as the output: "hello" and "world." In the `outputs` section of the job, we map these step outputs to job outputs called: `output1` and `output2`. In the `on.workflow_call.outputs` section we then define two outputs for the workflow itself, one called `firstword` which we map to `output1`, and one called `secondword` which we map to `output2`.  

The `value` must be set to the value of a job-level output within the called workflow. Step-level outputs must first be mapped to job-level outputs as shown below.

For more information, see "[AUTOTITLE](/actions/using-jobs/defining-outputs-for-jobs#overview)" and "[For more information about setting outputs with `jobs.<job_id>.outputs`, see "[AUTOTITLE](/actions/using-jobs/defining-outputs-for-jobs#overview)."](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_calloutputs)"
```